src/site/stages/build/process-cms-exports/tests/transformed-entities/node-event_listing.json

```diff
@@ -0,0 +1,68 @@
+{
+  "contentModelType": "node-event_listing",
+  "entityType": "node",
+  "entityBundle": "event_listing",
+  "entityUrl": {
+    "breadcrumb": [],
+    "path": "/outreach-and-events/events"
+  },
+  "entityMetatags": [
+    {
+      "__typename": "MetaValue",
+      "key": "title",
+      "value": "Outreach & Events | Outreach Events | Veterans Affairs"
+    },
+    {
+      "__typename": "MetaValue",
+      "key": "twitter:card",
+      "value": "summary_large_image"
+    },
+    {
+      "__typename": "MetaProperty",
+      "key": "og:site_name",
+      "value": "Veterans Affairs"
+    },
+    {
+      "__typename": "MetaValue",
+      "key": "twitter:description",
+      "value": "Outreach events"
+    },
+    {
+      "__typename": "MetaValue",
+      "key": "description",
+      "value": "Outreach events"
+    },
+    {
+      "__typename": "MetaValue",
+      "key": "twitter:title",
+      "value": "Outreach events | Veterans Affairs"
+    },
+    {
+      "__typename": "MetaValue",
+      "key": "twitter:site",
+      "value": "@DeptVetAffairs"
+    },
+    {
+      "__typename": "MetaProperty",
+      "key": "og:title",
+      "value": "Outreach events | Veterans Affairs"
+    },
+    {
+      "__typename": "MetaProperty",
+      "key": "og:description",
+      "value": "Outreach events"
+    }
+  ],
+  "title": "Outreach events",
+  "changed": 1565895173,
+  "metatag": null,
+  "fieldIntroText": "Veterans' benefits help Veterans and their families buy homes, earn degrees, start careers, stay healthy, and more. Join an event for conversation and information.",
+  "fieldOffice": {
+    "contentModelType": "node-office",
+    "entity": {
+      "entityType": "node",
+      "entityBundle": "office",
+      "entityLabel": "Outreach and events"
+    }
```

Comment:
I took a peek at `pages.json` and saw that it had a `reverseFieldOfficeNode` in the fieldOffice.entity.  We may have to do some investigation around `node-field_office`

Reply:
Ooh, good call. Where'd this come from, @rianfowler? This is what I'm seeing in the `pages.json`: 
![image](https://user-images.githubusercontent.com/12970166/73021429-70a23700-3ddc-11ea-87e6-00528943b632.png)

Reply:
dunno- AFAIK i didn't edit that file. I copied it right out of pages.json.

Reply:
Brooks and I looked into this and found the following conflict: 

pages.json doesn't reference another node 

```
...
"fieldOffice": {
            "targetId": 735,
            "entity": {
              "reverseFieldOfficeNode": {
                "entities": [
                  {},
                  {},
                  {},
...
]}}}
```

tome sync does in node.30295cab-89be-4173-a33c-8c0ca0a85d07.json:

```
    ],
    "field_office": [
        {
            "target_type": "node",
            "target_uuid": "c71e29a0-431f-44a5-ac5c-cee0c0ef0ceb"
        }
```

Reply:
Fun fact: There are two instances of `reverseFieldOfficeNode`s in the `pages.json` we have in that dump. Both of them have `entities` arrays that are filled with all blank objects `{}`. :unamused: